CockpitPlugin/src/main/java/com/polidea/cockpitplugin/generator/DebugCockpitGenerator.kt

```diff
@@ -10,12 +10,12 @@ import javax.lang.model.element.Modifier
 
 internal class DebugCockpitGenerator : BaseCockpitGenerator() {
 
-    private var shouldGenerateColorFields = false
-
     override fun generate(params: List<CockpitParam<*>>, file: File?) {
         val propertyMethods = params.fold(mutableListOf<MethodSpec>()) { acc, param ->
             acc.apply {
-                when (param.value) {
+                val paramName = param.name
+                val value = param.value
```

Comment:
We have "paramName" for param name, but just "value" for param value. Let's make it consistent. :)